tlcc_enclave/generate_protos.sh

```diff
@@ -9,10 +9,27 @@ PROTOC_OPTS="--plugin=protoc-gen-nanopb=$NANOPB_PATH/generator/protoc-gen-nanopb
 
 FABRIC_PROTOS=$FABRIC_PATH/protos
 
-BUILD_DIR=enclave/protos
+if [ "$1" != "" ]; then
+    BUILD_DIR=$1
+else
+    BUILD_DIR=enclave/protos
+fi
 rm -rf $BUILD_DIR
 mkdir -p $BUILD_DIR
 
+
```

Comment:
as discussed, i think we can save ourself from this copying but in any case nice to remove having user to do it manually. You should though also update docu correspondingly

Reply:
agree; this is PR we keep copying files but there will be an extra PR addressing this issue. (This PR here is already big enough)